distribution/ddf-common/src/main/resources/security/default.policy

```diff
@@ -126,12 +126,18 @@ grant
     permission java.util.PropertyPermission "javax.net.ssl.*", "read";
 
     permission java.io.FilePermission "${ddf.home}${/}bin_third_party${/}-", "read, execute";
+
+    permission java.io.FilePermission "<<ALL FILES>>", "execute";
 };
 
 grant {
     // User's home directory
     permission java.io.FilePermission "${user.home}${/}-", "read, write";
 
+    // Schema and Schematron directories
+    permission java.io.FilePermission "${ddf.home}${/}schema${/}-", "read";
+    permission java.io.FilePermission "${ddf.home}${/}schematron${/}-", "read";
+
```

Comment:
I think your suggestion of putting them under _home/data_ or _home/ext_ was good. I feel like these directories shouldn't  be hanging off _home_.